src/lib/types.ts

```diff
@@ -524,6 +524,7 @@ export const enum NodeName {
 export const enum StateProp {
   errorHandlers = 'error',
   loadHandlers = 'load',
+  src = 'src',
```

Comment:
Is this guy ok or should it be a number?

Reply:
A number would be fine.

Reply:
Thanks, just updated it :)